calblueprint/meepanyar-node
# Changes to deploy backend to heroku

This PR contains all the necessary changes needed to deploy meepanyar-node server to heroku. Sorry for the mish-mash of changes.

Changelist:
- Removed CORs middleware so that Airlock handles CORs on its own
- Upgraded Airlock so that it has `sameSite` attribute when it sets cookies (allows setting cookie over CORs)
- Added `PRODUCTION_WEB_URL` environment variable for deployment (can be ignored for development)

How to test:
1. Go to: https://meepanyar.netlify.app/ and make sure you can successfully log in (the first login might take a while because heroku needs to wake up the backend)
2. Run the backend and make sure you can still use the frontend locally (login via localhost:3000)
3. Make sure requests are still able to be made to the backend (create a customer and make sure the customer is actually created and there are no CORs errors)
